lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -170,6 +170,7 @@
     <datatype extension="hivtrace" type="galaxy.datatypes.text:Json" mimetype="application/json" subclass="true" display_in_upload="false"/>
     <datatype extension="cool" type="galaxy.datatypes.binary:Cool" mimetype="application/octet-stream" display_in_upload="true"/>
     <datatype extension="mcool" type="galaxy.datatypes.binary:MCool" mimetype="application/octet-stream" display_in_upload="true"/>
+    <datatype extension="h5_model" type="galaxy.datatypes.binary:H5_Model" mimetype="application/octet-stream" display_in_upload="true"/>
```

Comment:
@qiagu my concern is that this name is too generic. Can we not name it gxmlm or gxml_model or something like that.

Reply:
How about `h5mlm`? `h5` is more specific.

Reply:
Also ok :)